jobs/release/prerelease.yml

```diff
@@ -29,6 +29,9 @@ extends:
         templateContext:
           type: releaseJob
           isProduction: true
+          sdl:
+            codeSignValidation:
+              additionalTargetsGlobPattern: -|*.js
```

Comment:
I think you should use the **\\*.js pattern instead of *.js, since that's the pattern that Code Sign Validation uses to include .js files by default

Reply:
Updated.